### PR TITLE
test: update CMake in Dockerfile's template

### LINKED ISSUE #1046: tt install: test fails after tarantool 3.3 release
The test `test_install_tarantool_in_docker` fails after Tarantool 3.3 release. We need to find a reason and fix it.

### PR DESCRIPTION
**Pull request by dmyger**
It required to build Tarantool 3.3 in Docker.

Closes #1046

### CODE REVIEW

File: test/integration/install/test_install.py [hunk 1]
```diff
@@ -360,7 +360,7 @@ def test_install_tarantool_in_docker(tt_cmd, tmp_path):
     # Check tarantool glibc version.
     out = subprocess.getoutput("objdump -T " + os.path.join(tmp_path, "bin", "tarantool") +
                                " | grep -o -E 'GLIBC_[.0-9]+' | sort -V | tail -n1")
-    assert out == "GLIBC_2.27"
+    assert out == "GLIBC_2.34"
```
Comment (dmyger) [inline test/integration/install/test_install.py:363]: It does not clear, why we need checks `glibc` version of `Tarantool`? What depends on it?

Comment (oleg-jukovec) [reply test/integration/install/test_install.py:363]: I don't know, so let just update it and fix a problem later.

**Comment by oleg-jukovec** [comment]
`test: Update Ubuntu in Dockerfile's template` -> `test: update Ubuntu in Dockerfile's template`

**Review by oleg-jukovec — COMMENTED** [review]
Please, fix the commit title:

"test: Update Ubuntu in Dockerfile's template" -> 
"test: update Ubuntu in Dockerfile's template"

**Comment by dmyger** [comment]
> since it changes a tt pack behavior.

No, `pack` uses own Dockerfile template: `Dockerfile.pack.build`.
Only the case of `tt install --use-docker` is considered here.

**Comment by dmyger** [comment]
> fix the commit title

Done: "test: update CMake in Dockerfile's template"

**Review by oleg-jukovec — APPROVED** [review]

**Review by patapenka-alexey — APPROVED** [review]
Thanks for the patch. LGTM!

**Pull request merged**